network-api/networkapi/wagtailpages/static/js/directory-listing-filters.js

```diff
@@ -105,11 +105,13 @@
     });
 
     // And then we update the content that the user sees:
+    profileContainer.style.height = null;
```

Comment:
setting to `null` is a little tricky, better to use this:
```suggestion
    profileContainer.style.removeProperty(`height`);
```

Reply:
great suggestion 👍